language/move-lang/src/expansion/ast.rs

```diff
@@ -373,6 +445,20 @@ impl AstDebug for FunctionSignature {
     }
 }
 
+impl AstDebug for SpecCondition_ {
+    fn ast_debug(&self, w: &mut AstWriter) {
+        match self.kind {
+            SpecConditionKind::AbortsIf => {
+                w.write("aborts_if ");
```

Comment:
nit, no semi-colon and you won't need the braces. A bit easier to read

Reply:
Done